sentry.gradle

```diff
@@ -169,7 +202,21 @@ gradle.projectsEvaluated {
             group = 'sentry.io'
 
             delete sourcemapOutput
-            delete "$buildDir/intermediates/assets/release/index.android.bundle.map" // react native default bundle dir
+            delete "$buildDir/intermediates/assets/release/index.android.bundle.map"
+            // react native default bundle dir
+        }
+
+        def modulesCleanUpTask = tasks.create(name: nameModulesCleanup, type: Delete) {
+            description = "clean up collected modules generated file"
+            group = 'sentry.io'
+
+            delete modulesOutput
+        }
+
+        def packageTasks = tasks.findAll { task -> task.name == 'packageRelease' && task.enabled }
```

Comment:
H: You can't rely on `packageRelease`, you have to use `bundleTask.name` as part of the name, as shared on the Android Gradle plugin code.

Reply:
But the bundle task is about creating a JS bundle and I need to wait for packing.

Reply:
That's fine but it's still a problem, you have to compose the task name with the variant name, if you don't do that, an Android app with `flavors` or custom `buildTypes` won't work, you'd be depending on the wrong task.

Reply:
Ohh, I see, thank you 🙏, I'll fix that.